### DISCUSSION TITLE
求 冰风谷 2.5版本 我这里有汉化补丁

### DISCUSSION THREAD

**Discussion by github342655596** [Gamuxorg/bbs#247] [投递和请求游戏]
游戏名称：冰风谷增强版
游戏名称(英文)：Icewind Dale Enhanced Edition
下载地址 :（如果有的话）：汉化补丁链接: https://pan.baidu.com/s/1JUSP7q28h42F_QxAvpfrxw  密码: 2ll1
游戏站里面那个太旧了。

**Comment by shouhuanxiaoji** [comment]
em.. 现有的版本有汉化没？

**Comment by wangling12** [comment]
你好，汉化补丁的链接失效了，可否再发一下

**Comment by wangling12** [comment]
已上传
